mne/report.py

```diff
@@ -316,6 +316,35 @@ def _update_html(html, report_fname, report_sectionlabel):
     return htmls, report_fnames, report_sectionlabels
 
 
+def open_report(fname, **params):
+    """Read a saved report or, if it doesn't exist yet, create a new one.
+
+    The returned report is used as a context manager and any changes to the
+    report are saved when exiting the context block.
+
+    Parameters
+    ----------
+    fname : str
+        The file containing the report, stored in the HDF5 format. If the file
+        does not exist yet, a new report is created that will be saved to the
+        specified file.
+    **params : list of parameters
+        Parameters to use when creating a new Report object.
+
+    Returns
+    -------
+    report : instance of Report
+        The report.
+    """
+    if op.exists(fname):
+        # Don't pass **params to the Report
```

Comment:
Instead of totally ignoring these entries it might be worth checking that they are correct for the loaded `Report`, at least for the `params` entries that you can (like `subject`, `subjects_dir`, etc.). It should save users from making some mistakes.

Reply:
you got it.